From 95ce588e21398d04a1f50fb7cc226236b0b725b5 Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@samba.org>
Date: Thu, 14 Feb 2013 12:14:33 +0100
Subject: [PATCH] waf: Fix correct linking of libreplace with
 cmdline-credentials.

cmdline-credentials needs rep_getpass from libreplace. The function has
been replaced in master so this issue only exists in 4.0 releases.

Reviewed-by: Andrew Bartlet <abartlet@samba.org>
Signed-off-by: Andreas Schneider <asn@samba.org>
---
 source4/lib/cmdline/wscript_build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/source4/lib/cmdline/wscript_build b/source4/lib/cmdline/wscript_build
index 9b6f6e5..eec92d4 100644
--- a/source4/lib/cmdline/wscript_build
+++ b/source4/lib/cmdline/wscript_build
@@ -4,6 +4,7 @@ bld.SAMBA_LIBRARY('cmdline-credentials',
                   source='credentials.c',
                   autoproto='credentials.h',
                   public_deps='samba-credentials popt',
+                  deps='replace',
                   private_library=True)
 
 bld.SAMBA_SUBSYSTEM('POPT_SAMBA',
-- 
1.8.1.2

